translations/locales/en.json

```diff
@@ -252,6 +252,10 @@
                     "escaping": {
                         "title": "Escaping",
                         "content": "<p>If you want to use a special Markdown character in your document (such as displaying literal asterisks), you can escape the character with the backslash (<code>\\\\</code>). Markdown will ignore the character directly after a backslash.</p>\n"
+                    },
+                    "shortcuts": {
+                        "title": "Shortcuts",
+                        "content": "<p>Press <code>Ctrl-S</code> to commit your changes. <br/> Press <code>Ctrl-Enter</code> to toggle between editing and previewing.</p>\n"
```

Comment:
So these translations are generated from Transifex by running the `gulp translations` task, and as soon as we publish a new translation, say someone contributes a new language and we need to deploy it, then your text here would get wiped out.

@mikemorris am I right in thinking that if we went in and changed the english phrasing of this in Transifex, then all the other language translations would be out of sync? Not a huge deal for this but definitely seems to be a pain if for example we need to change bigger chunks of critical text?